articles/getting-started/install-configure-bonsai.md

```diff
@@ -0,0 +1,86 @@
+---
+uid: BonsaiInstallationAndConfiguration
+title: Bonsai Installation and Configuration
+---
+
+# Bonsai Installation and Configuration
+
+## Install Bonsai
+
+To download Bonsai, select between the portable download and the installer download [here](https://bonsai-rx.org/docs/articles/installation.html).
+
+*   The **Portable** download installs a self-contained sandboxed Bonsai. Portable environments enable users to switch between different environments to prevent package conflicts or confusion between similar packages
```

Comment:
"self-contained sandboxed" is redundant, either one is valid though.

Reply:
Make sure that all sentences end in a period (I think that's the convention we agreed upon)

Reply:
got it